src/app/projects/states/dashboard/directives/task-dashboard/directives/task-description-card/task-description-card.tpl.html

```diff
@@ -19,9 +19,9 @@ <h5>Key Information</h5>
         You should aim to have this task completed by <strong>{{taskDef.target_date | humanizedDate}}</strong>.
       </li>
       <li ng-show="taskDef.due_date">
-        <strong>Due Date</strong>
+        <strong>Deadline</strong>
         &mdash;
-        You must complete this task by <strong>{{taskDef.due_date | humanizedDate}}</strong> or your tutor may not assess your work.
+        You must complete this task by <strong>{{taskDef.due_date | humanizedDate}}</strong>. Submitting after this date means your tutor will not provide feedback for your submission. It will be checked in your portfolio.
```

Comment:
"Submitting after this date means your tutor will not provide feedback for your submission, but that it will be checked in your portfolio."

Reply:
👍